Source/Scene/GltfBufferViewLoader.js

```diff
@@ -19,7 +19,7 @@ import ResourceLoaderState from "./ResourceLoaderState.js";
  * @param {ResourceCache} options.resourceCache The {@link ResourceCache} (to avoid circular dependencies).
  * @param {Object} options.gltf The glTF JSON.
  * @param {Number} options.bufferViewId The buffer view ID.
- * @param {Resource} options.gltfResource The {@link Resource} pointing to the glTF file.
+ * @param {Resource} options.gltfResource The {@link Resource} containing the glTF.
  * @param {Resource} options.baseResource The {@link Resource} that paths in the glTF JSON are relative to.
```

Comment:
Just to clarify, under what circumstances would `gltfResource` differ from `baseResource`?

Reply:
Usually they are the same but the user can set their own base path in the Model API like [here](https://github.com/CesiumGS/cesium/blob/7cb225dc35a5f48424b9e7f7123aa6c532297b67/Source/Scene/Model.js#L192)